samples/subsys/usb/uvc/README.rst

```diff
@@ -0,0 +1,143 @@
+.. zephyr:code-sample:: uvc
+   :name: USB Video sample
+   :relevant-api: usbd_api video_interface
+
+   USB Video sample sending frames over USB.
+
+Overview
+********
+
+This sample demonstrates how to use an USB Video Class instance to
+send video data over USB.
+
+Upon connection, a video interface would show-up in the operating system,
+using the same protocol as most webcams, and be detected as such.
+
+Any software application to read and process video would then be able
+to access the stream.
+
+Requirements
+************
+
+USB is the only requirement for this sensor, as an emulated image
+sensor and MIPI receiver is used to provide a test pattern from
+software.
+
+The USB descriptor configuration is extracted from these drivers, and
+the user does not need to provide them.
+
+Building and Running
+********************
+
+Build the sample application and flash the resulting binaries.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/subsys/usb/uvc
+   :board: nrf52840dongle
+   :goals: build flash
+   :compact:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/subsys/usb/uvc
+   :board: mini_stm32h734
+   :goals: build flash
+   :compact:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/subsys/usb/uvc
+   :board: rpi_pico
+   :goals: build flash
+   :compact:
+
+Upon reboot, the device is expected to be detected by the operating
+system.
+
+On Linux, OSX and BSDs this is visible using the ``dmesg`` and ``lsusb`` commands.
```

Comment:
I think this is not needed to be documented as it depends on OS (many OSes to be listed) and the command tool one would use (e.g. lsusb, usbview, etc.)

Reply:
The [Get Started](https://docs.zephyrproject.org/latest/develop/getting_started/index.html) guide is also OS (Windows/MacOSX/Ubuntu) and tool dependent (`pip` selected instead of `pipx` or `pdm`). Maybe I can target these 3 platforms in the doc, letting users of other platforms decide on their equivalent command.

I will try a tab-based view like the get-started doc and ask your feedback on the result.

Reply:
I think the "Get Started" guide is a different case as it documented how to install Zephyr (which is "unique") on different OSes. Here, _if_ we say this is an "ordinary" USB / UVC device, it should be like other USB devices so my point was just wondering if it is better to document it in a "refactored way". But as you pointed out in other comment, it is a bit different here so tha's fine with me as it is currently done.

Reply:
I tried a different approach:
- very minimal "crash course" to check if it worked or debug if it did not, but only point at what to try succinctly
- be more specific to this particular sample: which VID:PID to expect, name of the video interfaces directly.
- only use the 3 OSes supported by Zephyr. The commands for other OS are likely possible to deduce out of this

You are right that it is good to let user know this is a regular webcam as they are used to.
Let me know if it might still be possible to do better. :)